xml/System.DirectoryServices.AccountManagement/PrincipalCollection.xml

```diff
@@ -316,7 +316,7 @@
   
 |Exception|Description|  
 |---------------|-----------------|  
-|<xref:System.InvalidOperationException>|If this method is called on the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> PrincipalCollection, the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> property cannot be cleared because one or more of the group's members have this group as their primary group.|  
+|<xref:System.InvalidOperation>|If this method is called on the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> PrincipalCollection, the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> property cannot be cleared because one or more of the group's members have this group as their primary group.|  
```

Comment:
😕

Reply:
I'll be reviewing this PR @NextTurn and only bringing the files with changes that we really want to bring.